src/views/preview/preview.scss

```diff
@@ -174,14 +170,13 @@ $stage-height: 404px;
 
     .guiPlayer {
         display: inline-block;
-        margin-left: -.5rem;
```

Comment:
here's where we remove the margin hack! (hard to see because of all the whitespace changes)